arcgis-ios-sdk-samples/Utility Network/Display utility association/DisplayUtilityAssociationVC.swift

```diff
@@ -0,0 +1,141 @@
+                let connectivityValue = AGSUniqueValue(description: "Connectivity", label: "", symbol: self.connectivitySymbol, values: [AGSUtilityAssociationType.connectivity])
+                self.associationsOverlay.renderer = AGSUniqueValueRenderer(fieldNames: ["AssociationType"], uniqueValues: [attachmentValue, connectivityValue], defaultLabel: "", defaultSymbol: nil)
+                
+                // Populate the legened.
+                self.attachmentSymbol.createSwatch(withBackgroundColor: nil, screen: .main) { [weak self] (image, error) in
+                    if let error = error {
+                        print("Error creating swatch: \(error)")
+                    } else {
+                        self?.attachmentBBI.image = image?.withRenderingMode(.alwaysOriginal)
+                    }
+                }
+                self.connectivitySymbol.createSwatch(withBackgroundColor: nil, screen: .main) { [weak self] (image, error) in
+                    if let error = error {
+                        print("Error creating swatch: \(error)")
+                    } else {
+                        self?.connectivityBBI.image = image?.withRenderingMode(.alwaysOriginal)
+                    }
+                }
+                self.addAssociationGraphics()
+                self.mapViewDidChange()
+            }
+        }
+    }
+    
+    func addAssociationGraphics() {
+        // Check if the current viewpoint is outside of the max scale.
+        if let targetScale = mapView.currentViewpoint(with: .centerAndScale)?.targetScale {
+            if targetScale >= maxScale {
+                return
+            }
+        }
+        
+        // Check if the current viewpoint has an extent.
+        if let extent = mapView.currentViewpoint(with: .boundingGeometry)?.targetGeometry.extent {
+            //Get all of the associations in extent of the viewpoint.
+            utilityNetwork.associations(withExtent: extent) { [weak self] (associations, error) in
+                if let error = error {
+                    print("Error loading associations: \(error)")
+                } else {
+                    guard let self = self else { return }
+                    associations?.forEach { association in
+                        // Check if the graphics overlay already contains the association.
+                        let graphics = self.associationsOverlay.graphics as! [AGSGraphic]
+                        let associationGID = association.globalID
+                        let existingAssociations = graphics.filter { $0.attributes["GlobalId"] as! UUID == associationGID }
+                        
+                        // If it the current association does not exist, add it to the graphics overlay.
+                        if existingAssociations.isEmpty {
+                            var symbol = AGSSymbol()
```

Comment:
Please do not assign a value here.

```suggestion
                            let symbol: AGSSymbol
```